addons.yaml

```diff
@@ -207,3 +207,11 @@ microk8s-addons:
       supported_architectures:
         - amd64
         - arm64
+
+    - name: "kubearmor"
+      description: "Cloud-native runtime security enforcement system for k8s"
+      version: "0.10.2"
+      check_status: "daemonset.apps/kubearmor"
```

Comment:
Please add `      confinement: "classic"` as this addon would not work in strict.